src/node_options-inl.h

```diff
@@ -93,6 +93,23 @@ void OptionsParser<Options>::AddOption(
   });
 }
 
+template <typename Options>
+void OptionsParser<Options>::AddOption(
```

Comment:
Can you move this to a different pull request?

Reply:
Why? It's directly needed for this change.

Reply:
This pull request didn't attract enough reviewers qualified to review this particular change. I don't feel comfortable with approving a change on a part of Node.js that I didn't work on.

Reply:
I'm stilly very against splitting this up. I think that will confuse reviewers, as adding a new feature without cause/reason doesn't make much sense.

> This pull request didn't attract enough reviewers qualified to review this particular change

I can CC `cpp-reviewers`? 

I'd rather get more vis on this PR than open a new one?